spec/puppetserver/ca/config/puppet_spec.rb

```diff
@@ -69,7 +71,7 @@
       conf.load
 
       expect(conf.errors).to be_empty
-      expect(conf.settings[:cacert]).to eq('/foo/bar/ca/ca_crt.pem')
+      expect(conf.settings[:localcacert]).to eq('/foo/bar/certs/ca.pem')
```

Comment:
What's the reason for this change?

Reply:
Justin also made this change in his original PR for this on the other branch: the problem is that the location of `cacrt` becomes unpredictable with this change, without a bunch of mocking, but this is testing verifying that we pick up the customer `ssldir` location. So this change switches to testing a setting that still interpolates that.

Reply:
ahh gotcha, thanks